os/net/netmgr/netstack_lwip.c

```diff
@@ -427,22 +427,24 @@ static void _get_raw_info(struct netmon_sock *sock_info, struct lwip_sock *lsock
 	sock_info->remote.ip.sin_port = 0;
 }
 
-extern struct lwip_sock *get_lwip_sock_info(void);
+extern struct socketlist *get_lwip_sock_info(void);
```

Comment:
socketlist is for TizenRT, lwip_sock is for lwip. get_lwip_sock_info() is implemented in sockets.c which lwip contains. so you shouldn't use socketlist in lwip code. it violates layered dependency.
you'd better to call sched_getsockets() which TizenRT already provides instead of get_lwip_sock_info

Reply:
kept in mind. will change it.

Reply:
done.